java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -179,6 +179,109 @@ public static int getNullCount(final ArrowBuf validityBuffer, final int valueCou
     return 8 * sizeInBytes - count;
   }
 
+  /**
+   * Tests if all bits in a validity buffer are 0.
+   * @param validityBuffer the validity buffer.
+   * @param valueCount the bit count.
+   * @return true if all bits are 0, and false otherwise.
+   */
+  public static boolean allBitsNull(final ArrowBuf validityBuffer, final int valueCount) {
```

Comment:
can you please add a benchmark for this ? I'm curious if a simple loop that checks each byte performs any better.

Reply:
@pravindra thanks for the good suggestion. 
I have added a benchmark for the method. The results are as follows:

check by bit (current implementation)
BitVectorHelperBenchmarks.allBitsNullBenchmark  avgt    5  1472.140 ± 13.670  ns/op

check by byte
BitVectorHelperBenchmarks.allBitsNullBenchmark  avgt    5  139.718 ± 0.268  ns/op

check by long (implementation introduced by this PR)
BitVectorHelperBenchmarks.allBitsNullBenchmark   avgt    5  26.445 ± 0.097  ns/op

Reply:
thanks @liyafan82